src/layer/__tests__/fixtures.findTilesS2L1C.ts

```diff
@@ -0,0 +1,325 @@
+                ],
+              ],
+            ],
+          },
+          'eo:cloud_cover': 0.08,
+          'eo:bands': [
+            { name: 'B01', common_name: 'coastal', center_wavelength: 0.4427, full_width_half_max: 0.021 },
+            { name: 'B02', common_name: 'blue', center_wavelength: 0.4924, full_width_half_max: 0.066 },
+            { name: 'B03', common_name: 'green', center_wavelength: 0.5598, full_width_half_max: 0.036 },
+            { name: 'B04', common_name: 'red', center_wavelength: 0.6646, full_width_half_max: 0.031 },
+            { name: 'B05', center_wavelength: 0.7041, full_width_half_max: 0.015 },
+            { name: 'B06', center_wavelength: 0.7405, full_width_half_max: 0.015 },
+            { name: 'B07', center_wavelength: 0.7828, full_width_half_max: 0.02 },
+            { name: 'B08', common_name: 'nir', center_wavelength: 0.8328, full_width_half_max: 0.106 },
+            { name: 'B8A', common_name: 'nir08', center_wavelength: 0.8647, full_width_half_max: 0.021 },
+            { name: 'B09', common_name: 'nir09', center_wavelength: 0.9451, full_width_half_max: 0.02 },
+            { name: 'B10', common_name: 'cirrus', center_wavelength: 1.3735, full_width_half_max: 0.031 },
+            { name: 'B11', common_name: 'swir16', center_wavelength: 1.6137, full_width_half_max: 0.091 },
+            { name: 'B12', common_name: 'swir22', center_wavelength: 2.2024, full_width_half_max: 0.175 },
+          ],
+          'eo:gsd': 10,
+          'proj:bbox': [696896.5684521495, 4586375.95284284, 814572.5734874466, 4704040.517882267],
+          platform: 'sentinel-2a',
+        },
+        links: [
+          {
+            href:
+              'https://services.sentinel-hub.com/api/v1/catalog/collections/sentinel-2-l1c/items/S2A_MSIL1C_20200823T100031_N0209_R122_T32TQM_20200823T121427',
+            rel: 'self',
+            type: 'application/json',
+          },
+          {
+            href: 'https://services.sentinel-hub.com/api/v1/catalog/collections/sentinel-2-l1c',
+            rel: 'parent',
+          },
+          {
+            href:
+              "https://scihub.copernicus.eu/dhus/odata/v1/Products('966aba13-b34f-4563-85ae-356503fb6c7b')/$value",
+            rel: 'derived_from',
+            title: 'scihub download',
+          },
+        ],
+        assets: {
+          thumbnail: {
+            href: 'https://roda.sentinel-hub.com/sentinel-s2-l1c/tiles/32/T/QM/2020/8/23/0/preview.jpg',
+            title: 'thumbnail',
+            type: 'image/png',
+          },
+          data: {
+            href: 's3://sentinel-s2-l1c/tiles/32/T/QM/2020/8/23/0/',
+            title: 's3',
+            type: 'inode/directory',
+          },
+        },
+      },
+    ],
+    links: [
+      {
+        href: 'https://services.sentinel-hub.com/api/v1/catalog/search',
+        rel: 'self',
+        type: 'application/json',
+      },
+    ],
+    context: { limit: 50, returned: 3 },
+  };
+  /* eslint-enable */
```

Comment:
This should probably be the end of the block? (not that it matters much)

Reply:
I wanted to disable lint just for that particular response (snake_case instead of camelCase)

Reply:
I know, sorry if I wasn't clear - the block starts with `/* eslint-enable */` (correct) and ends with `/* eslint-enable */` (wrong, should be `/* eslint-disable */`).

I am assuming here that you wanted to disable linting just for this block, of course. If not, I would suggest putting `/* eslint-disable */` at the end of the file to clearly communicate with the reader that this was intentional (and then one of the `enable`s is not needed).